tests/phpunit/tests/term/wpGetObjectTerms.php

```diff
@@ -614,6 +614,7 @@ public function test_parent_0() {
 
 	/**
 	 * @ticket 10142
+	 * @ticket 57701: Updated test to reflect that wp_get_object_terms does not prime the cache.
 	 */
 	public function test_termmeta_cache_should_be_lazy_loaded_by_default() {
```

Comment:
Let's change the name of this test as well.

Reply:
Anything specific you have in mind here as a replacement name?

Reply:
`test_termmeta_cache_not_should_be_lazy_loaded_by_default`

Reply:
done